stubs/python-decouple/decouple.pyi

```diff
@@ -0,0 +1,98 @@
+from _typeshed import Incomplete
+from collections import OrderedDict
+from collections.abc import Callable, Sequence
+from configparser import ConfigParser
+from typing import Any, Generic, TextIO, TypeVar, overload
+
+_T = TypeVar("_T")
+_TCsv = TypeVar("_TCsv", bound=Sequence[Any])
```

Comment:
I think this upper bound is too strict, I would change this to `Collection[Any]` at the very least, but technically this can be anything and the docs also state as much (you could e.g. imagine someone calculating an integer hash from the individual values by providing `post_process=lambda x: hash(tuple(x))`) so I'm not sure if it's worth providing an upper bound.

Reply:
Yep, that's a great point!